execution-environment.yml

```diff
@@ -38,6 +38,7 @@ dependencies:
     epel-release [platform:rpm]
     python-unversioned-command [platform:rpm]
     unzip [platform:rpm]
+    podman-remote [platform:rpm]
```

Comment:
i think this is just a pip install and not rpm

Reply:
It can be installed as an rpm, but will change it to use pip

Reply:
Switched back to rpm because of - 
```
63.70 ERROR: Ignored the following versions that require a different python version: 3.0.0 Requires-Python <3
63.70 ERROR: Could not find a version that satisfies the requirement podman_remote (from versions: none)
63.70 ERROR: No matching distribution found for podman_remote
```